package.json

```diff
@@ -41,11 +41,11 @@
 		"playwright": "^1.45.1",
 		"react": "^18.3.1",
 		"react-dom": "^18.3.1",
-		"react-router": "0.0.0-nightly-90ebbf91d-20240718"
+		"react-router": "0.0.0-experimental-49eef6a01"
```

Comment:
This is an experimental release from https://github.com/remix-run/react-router/pull/11833 that fixes the issues we found on the stream:

* `__manifest` is now pre-rendered
* Trailing slashes are removed from single fetch data calls so no more invalid `/about/.data` calls
* Hydration loops should be resolved since we detect via matches and not a direct URL comparison